### DISCUSSION TITLE
老学长的建议 - HAUE CS WIKI

### DISCUSSION THREAD

**Discussion by giscus** [Doge2077/haue-cs-wiki#15] [General]
# 老学长的建议 - HAUE CS WIKI

https://doge2077.github.io/haue-cs-wiki/%E6%96%B0%E6%89%8B%E5%85%A5%E9%97%A8/%E8%80%81%E5%AD%A6%E9%95%BF%E7%9A%84%E5%BB%BA%E8%AE%AE/

**Comment by Doge2077** [comment]
建议学长不要建议😅
